Mathlib/Algebra/GroupWithZero/Int.lean

```diff
@@ -12,19 +12,17 @@ import Mathlib.Algebra.GroupWithZero.WithZero
 
 assert_not_exists Ring
 
-local notation "ℤₘ₀" => WithZero (Multiplicative ℤ)
+open Multiplicative
 
 namespace WithZero
 
-open Multiplicative
-
-theorem ofAdd_zpow (a : ℤ) : (↑(ofAdd a) : ℤₘ₀) = ofAdd (1 : ℤ) ^ a := by
```

Comment:
Why are we deprecating this

Reply:
1. The name is definitely wrong.
2. It's the equivalent of `Algebra.algebraMap_eq_smul_one` I guess. Do you really care about keeping it?

Reply:
I guess 1 is a good enough reason.